nixos/tests/containers-tmpfs.nix

```diff
@@ -0,0 +1,45 @@
+# Test for NixOS' container support.
+
+import ./make-test.nix ({ pkgs, ...} : {
+  name = "containers-bridge";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ ckampka ];
+  };
+
+  machine =
+    { config, pkgs, ... }:
+    { imports = [ ../modules/installer/cd-dvd/channel.nix ];
+      virtualisation.writableStore = true;
+      virtualisation.memorySize = 768;
+
+      containers.webserver =
+        {
+          autoStart = true;
+          privateNetwork = true;
+          tmpfs = [ "/var" ];
```

Comment:
Updated the test as requested, it now validated three cases:
Normal mount (/var), nested mount (/var/log while /var) and mounting to a destination that does not exist before.